protocols/raft/src/main/java/io/atomix/protocols/raft/roles/LeaderRole.java

```diff
@@ -86,6 +87,7 @@
  * Leader state.
  */
 public final class LeaderRole extends ActiveRole {
+  private static final int MAX_PENDING_COMMANDS = 1000;
```

Comment:
10k? 1k seems a little small.

Reply:
1k per session, where we typically can have hundreds of sessions open at once

Reply:
Sgtm